pkg/runtime/local/local.go

```diff
@@ -90,6 +93,105 @@ func (r *Runtime) GetGadgetInfo(_ context.Context, desc gadgets.GadgetDesc, pars
 	return runDesc.GetGadgetInfo(pars, args)
 }
 
+func (r *Runtime) GetOCIGadgetInfo(gadgetCtx runtime.GadgetContext, params *params.Params, args []string) (*api.GadgetInfo, error) {
```

Comment:
What are `params` and `args` and how are they different from the information already contained in gadgetCtx?